packages/components/select/src/select.ts

```diff
@@ -224,6 +224,13 @@ export const SelectProps = buildProps({
     type: definePropType<Placement[]>(Array),
     default: ['bottom-start', 'top-start', 'right', 'left'],
   },
+  /**
+   * @description which element the selection dropdown appends to
+   */
+  appendTo: {
+    type: String,
+    default: undefined,
```

Comment:
The default value would implicitly be undefined. Is it intentional to explicitly declare it?

Reply:
Don't remember, probably no. I can remove it

Reply:
Removed